dagshub/data_engine/model/query_result.py

```
@@ -778,6 +783,60 @@ def _get_all_annotations(self, annotation_field: str) -> List[IRImageAnnotationB
                 annotations.extend(dp.metadata[annotation_field].annotations)
         return annotations
 
+    def _get_all_video_annotations(self, annotation_field: str) -> List[IRVideoBBoxFrameAnnotation]:
+        all_anns = self._get_all_annotations(annotation_field)
+        return [a for a in all_anns if isinstance(a, IRVideoBBoxFrameAnnotation)]
+
+    @staticmethod
+    def _annotations_to_sequences(
+        video_annotations: List[IRVideoBBoxFrameAnnotation],
+    ) -> List["IRVideoSequence"]:
```

Ruff (0.15.6)

[error] Undefined name `IRVideoSequence`

(F821)